README.md

```diff
@@ -45,6 +45,22 @@ calendar.init(today, nextYear.getTime())
     .inMode(RANGE);
 ```
 
+If you want to be able to select dates in not continuous range, initiate calendar with collection
+of dates:
+
+```java
+protected Collection<Date> mSelectableDates = new ArrayList<Date>() {};
+protected long[] timestampArray = new long[count];
```

Comment:
This code example is very strange. Protected fields. 'm'-prefix on name. A loop over data which we can't see.